metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-93.7200
\ No newline at end of file
+93.5700
```

Comment:
Asking for 2-dev OK for a slight slip here - I think the code removed was better tested than average :)